.github/workflows/buildtracker.yml

```diff
@@ -0,0 +1,33 @@
+# Workflow that only runs when a commit lands on master.
+
+name: 👷‍♀️
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  buildtracker:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: git clone
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - run: yarn --frozen-lockfile
+    - run: yarn build-all
+
+    - name: Store in buildtracker
```

Comment:
wdyt of just adding this to the basic workflow? You need to install node, run yarn and build for this to work, so it's already a heavy workflow.

#10549 reused basic.yml for the same reason.

Reply:
good call.

Reply:
> why is installing node 10 so heavy?  

dont know.. i see it taking a while today but in [previous master commits](https://github.com/GoogleChrome/lighthouse/actions?query=workflow%3A%F0%9F%92%A1%F0%9F%8F%A0+branch%3Amaster), that step takes <3s.